Xamarin.Forms.Platform.iOS/Renderers/ScrollViewRenderer.cs

```diff
@@ -286,6 +286,10 @@ void OnScrollToRequested(object sender, ScrollToRequestedEventArgs e)
 						break;
 				}
 			}
+			if(newOffset.X + ScrollView.Width > ContentSize.Width)
```

Comment:
Could you include a sample and an UITest (tapping a Button and checking a Label value for example)?.

Reply:
Hi @jsuarezruiz 
i added a test. If its not enough, please modify the test.
Thank you

Reply:
Thanks!. Could you rebase 5.0.0 branch to fix the conflicts?

Reply:
@jsuarezruiz 
Conflicts fixed.